templates/post_emod3d_merge_ts.sl.template

```diff
@@ -21,6 +21,7 @@ if [[ ! "$(ls -A {{lf_sim_dir}}/OutBin)" ]]; then
      tar -xvf {{sim_dir}}/submission.tar
```

Comment:
does it still need submission.tar?

Reply:
Currently the tar files are gzip compressed (extension will reflect that in the next push).
Options are to extract the entire archive, then tar the old and new files together into a new archive, or to remove compression and append them to the old archive.
Will go with option 1 unless someone says otherwise.

Reply:
Yeah, they don't need to be compressed. Binaries especially are already compressed and then the slurm files are so small the compression doesn't matter.